.evergreen/config.yml

```diff
@@ -2119,7 +2120,7 @@ tasks:
             PKG_CONFIG_PATH=$PKG_CONFIG_PATH \
               make build-kms-test
             echo "Building build-kms-test ... end"
-            LD_LIBRARY_PATH=./install/libmongocrypt/lib \
+            LD_LIBRARY_PATH=./install/libmongocrypt/lib64 \
```

Comment:
These will need to be updated to point back to /lib as well

Reply:
I think we can leave the `testgcpkms-variant` with rhel 8.7 , the reason it was failing is because one of the libmongocrypt paths didn't have lib64 in it. Is there another reason to make them debian?

Reply:
Ah, okay, let's see if it works now.

Reply:
```
[2023/09/12 00:07:09.722] panic: Expected error message to contain "unable to retrieve aws credentials", but got "mongocrypt error 1: Error in KMS response. HTTP status=400. Response...
```

It looks like the `EXPECT_ERROR` needs to be updated.